### PR TITLE
Bridged networking feature

### PR DESCRIPTION
**Pull request by bgandon**
This PR is extracted from #17, focusing only on the new “bridged networking” feature. The work has been rebased on top of #22.

Original PR description by @ships in #17:
> - you had a WIP branch on bridged networks topic which I have progressed further. it is working for me but may not be complete; please give feedback at your convenience.
> - right now, you cannot enable or create bridged networks -- so it is only useful in cases where the bridged interface already exists.

Plus, we've gathered here the documentation proposed by @aussielunix in aussielunix@4bc4844.

### CODE REVIEW

File: docs/networks-bridged.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+No Virtualbox changes should be needed.  
+The following variables should be set with the `internal_ip` set to a static IP on the LAN.
+
+```
+internal_ip: 192.168.43.252
+internal_gw: 192.168.43.3
+internal_cidr: 192.168.43.0/24
+outbound_network_name: NatNetwork
+network_device: en0
```
Comment (rkoster) [inline docs/networks-bridged.md:16]: Maybe add a note that this network_device name can be found using `VBoxManage list bridgedifs` . In my case on MacOS I had to use `network_device: "en0: Wi-Fi (Wireless)"`.

File: src/bosh-virtualbox-cpi/driver/ssh_runner_test.go [hunk 2]
```diff
@@ -13,19 +13,23 @@ import (
...
 )
 
 var _ = Describe("SSHRunner", func() {
 	BeforeEach(func() {
 		if len(testSSHRunnerUsername) == 0 {
 			Skip("SSHRunner tests are not configured")
 		}
+		if testSSHRunnerHost == "" {
+			testSSHRunnerHost = "127.0.0.1"
+		}
```
Comment (rkoster) [inline src/bosh-virtualbox-cpi/driver/ssh_runner_test.go:26]: Unrelated to this PR

File: src/bosh-virtualbox-cpi/driver/ssh_runner_test.go [hunk 3]
```diff
@@ -13,19 +13,23 @@ import (
...
 	})
 
 	Context("HomeDir", func() {
 		It("returns proper home directory", func() {
 			opts := SSHRunnerOpts{
-				Host:       "127.0.0.1",
+				Host:       testSSHRunnerHost,
```
Comment (rkoster) [inline src/bosh-virtualbox-cpi/driver/ssh_runner_test.go:32]: Unrelated to this PR

**Review by rkoster — CHANGES_REQUESTED** [review]
When trying out these changes I'm getting:
```
Deploying:
  Creating instance 'bosh/0':
    Creating VM:
      Creating vm with stemcell cid 'sc-3aec8f82-4493-4e13-63ef-6674f124c0b3':
        CPI 'create_vm' method responded with error: CmdError{"type":"Bosh::Clouds::CloudError","message":"Creating VM with agent ID '{{a2de8839-3e4f-4b05-78ea-06610d254e40}}': Configuring NIC
s: Expected to find network 'en0'","ok_to_retry":false}
````

Had to change `network_device: "en0: Wi-Fi (Wireless)"` to make it work. Maybe add some docs on how to find the right device name.

**Review by ramonskie — APPROVED** [review]

**Pull request merged**